common/utils.go

```diff
@@ -1565,3 +1566,12 @@ func IsHTTPURL(str string) bool {
 
 	return regex.MatchString(str)
 }
+
+// GetDefaultContainerRuntime returns the preferred container runtime of the host
+func GetDefaultContainerRuntime() string {
+	_, err := exec.LookPath("podman")
+	if err == nil {
```

Comment:
Can we log a message at level info here that "Podman executable found in the directories named by the PATH environment variable. Using Podman as the container runtime.